client/src/offline-settings/mdn-worker.tsx

```diff
@@ -82,22 +34,19 @@ export class MDNWorker {
     }
   }
 
-  async autoUpdate() {
+  autoUpdate() {
     console.log("running auto update");
     if (this.timeout) {
       clearTimeout(this.timeout);
       this.timeout = null;
     }
-    await this.getUpdate();
+    this.checkForUpdate();
     this.update();
```

Comment:
`update()` depends on `checkForUpdate()` being done. That's why there was an await. Now if `autoUpdate()` is being call, for example by the timeout, it won't have the latest version in the db since `checkForUpdate()` runs async. We could add a flag to `checkForUpdate()` to also trigger the update when done.

Reply:
Since the `checkForUpdate()` implementation moved to SW and we just `postMessage` now, we cannot await here. **But** we could just `checkForUpdate()` as part of `updateContent()` in the SW, and then we can remove this call here.

Reply:
Resolved via c79fe3592.